test/wallet-http-test.js

```diff
@@ -1299,6 +1300,103 @@ describe('Wallet HTTP', function() {
     assert.equal(ns.info.name, name);
     assert.equal(ns.info.state, 'REVOKED');
   });
+
+  it('should require passphrase for auction TXs', async () => {
+    const passphrase = 'BitDNS!5353';
+    await wclient.createWallet('lockedWallet', {passphrase});
+    const lockedWallet = await wclient.wallet('lockedWallet');
+
+    // Fast-forward through the default 60-second unlock timeout
+    async function lock() {
```

Comment:
hm.. this async wont  have an effect (because you are not returning promise)
But if you want to wait for this fully you can just use normal `async/await`:

```js
  async function lock() {
    const wallet = await wdb.get('lockedWallet');
    await wallet.lock();
    // or
    return wallet.lock();
  }
 ```

Reply:
thanks